package/kernel/qca-ssdk/patches/104-enable-AQR-for-CPPE.patch

```diff
@@ -0,0 +1,12 @@
+Index: qca-ssdk-2024.06.13~c451136b/config
+===================================================================
+--- qca-ssdk-2024.06.13~c451136b.orig/config
++++ qca-ssdk-2024.06.13~c451136b/config
+@@ -291,6 +291,7 @@ else ifeq (HPPE, $(CHIP_TYPE))
+ 	IN_SFP=TRUE
+ 	IN_MALIBU_PHY=TRUE
+ else ifeq (CPPE, $(CHIP_TYPE))
++	IN_AQUANTIA_PHY=TRUE
```

Comment:
Instead of patching SSDK more, you can just pass this via the OpenWrt SSDK makefile for ipq60xx target